public/js/action-list.js

```diff
@@ -106,6 +122,18 @@
                 url = $list.attr('data-icinga-multiselect-url') + '?(' + filters.toArray().join('|') + ')';
             }
 
+            if (! footer.children('.selection-count').length) {
+                footer.prepend('<div class="selection-count"></div>');
+            }
+
+            var label = $list.data('icinga-multiselect-count-label').replace('%d', $activeItems.length);
+            var selectedItems = footer.find('.selection-count > .selected-items');
+            if (selectedItems.length) {
+                selectedItems.text(label);
+            } else {
+                footer.children('.selection-count').append('<span class="selected-items">' + label + '</span>');
+            }
+
```

Comment:
During testing I noticed that this runs also for non-multiselect lists. This has the side effect that if there's no native footer, line 126 prepends its HTML to an empty set. (`footer`) The set is empty, because the automatically added footer is only added in case it's a multiselect list.

Since it's useless anyway to show the selection count for non-multiselect lists, the entire block here should only run for multiselect lists. Then it's also guaranteed that a footer is found.